packages/react/src/Overlay/Overlay.tsx

```diff
@@ -155,6 +161,7 @@ const Overlay = React.forwardRef<HTMLDivElement, OwnOverlayProps>(
       preventFocusOnOpen,
       position,
       style: styleFromProps = {},
+      reflow = true,
```

Comment:
Should we make this prop opt-in by default? We could also see about forgoing the prop altogether. If we did the latter, it might be worth putting this under a feature flag.

Reply:
I'd say feature flag. Allowing reflow on small viewports seems like a logical default that you can choose to opt out of.

Minor nitpick: if we want to make reflow the default, I recommend we invert the prop to be `preventOverflow` so that you have to opt-out of the default.

Reply:
Oooh I like `preventOverflow`. Thank you for the suggestion too ✨